### PR TITLE
#949: Updated to slct 1.0.0

### PR DESCRIPTION
**Pull request by tomuben**
related to https://github.com/exasol/script-languages-release/issues/949

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,9 +19,9 @@ keywords = ['exasol', 'udf', 'script-languages']
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4"
-exasol-script-languages-container-tool = "^0.20.0"
-exasol-script-languages-container-ci = "^1.5.0"
-exasol-script-languages-container-ci-setup = "^1.4.0"
+exasol-script-languages-container-tool = "^1.0.0"
+exasol-script-languages-container-ci = "^1.6.0"
+exasol-script-languages-container-ci-setup = "^1.5.0"
 docker = "!= 7.0.0"
```
Comment (tkilias) [inline pyproject.toml:25]: This should not be necessary anymore

Comment (tkilias) [reply pyproject.toml:25]: Docker

Comment (tomuben) [reply pyproject.toml:24]: Removed `exasol-script-languages-container-tool`

Comment (tomuben) [reply pyproject.toml]: Done

Comment (tkilias) [reply pyproject.toml]: Please add exasol-script-languages-container-tool back, we added it intentionally

Comment (tomuben) [reply pyproject.toml]: Done

**Review by tkilias — APPROVED** [review]

**Pull request merged**